optuna/visualization/_optimization_history.py

```diff
@@ -1,11 +1,11 @@
 from __future__ import annotations
 
+from collections.abc import Callable
+from collections.abc import Sequence
 from enum import Enum
 import math
-from typing import Callable
 from typing import cast
 from typing import NamedTuple
```

Comment:
[Notes] `NamedTuple` is not listed in https://peps.python.org/pep-0585/#implementation, so we can keep it as it is.